include/aspect/postprocess/stress_component_statistics.h

```diff
@@ -0,0 +1,53 @@
+/*
+  Copyright (C) 2011 - 2022 by the authors of the ASPECT code.
```

Comment:
My earlier comment about this file has not been addressed yet:
My question:
> Is this file part of this PR or was it added in another one that was merged into this PR?

Your reply:
> It was part of this PR, but as discussed I will use it in a test and move it to tests/ or remove it completely.